<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>883</width>
        <height>622</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>883</width>
        <height>622</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>781</width>
          <height>521</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Список еды</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="foodTable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>400</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>859</width>
             <height>571</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="goToRecipesButton">
             <property name="text">
              <string>Список рецептов</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteFoodButton">
             <property name="text">
              <string>Удалить еду</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="goToAddFoodButton">
             <property name="text">
              <string>Добавить Еду</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="recipesPage">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>871</width>
          <height>591</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="availableRecipesLabel">
             <property name="text">
              <string>Можно приготовить</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lendRecipesLabel">
             <property name="text">
              <string>Можно одолжить еду</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="unavailableRecipesLabel">
             <property name="text">
              <string>Нельзя приготовить</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QListView" name="availableRecipeList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="lendRecipeList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="unavailableRecipeList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="goFromRecipesButton">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="recipeDetailPage">
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>89</x>
          <y>19</y>
          <width>691</width>
          <height>541</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="recipeNameLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="recipeDetailTable"/>
         </item>
         <item>
          <widget class="QPushButton" name="goToRecipesButton_2">
           <property name="text">
            <string>Назад</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="addFoodPage">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>238</x>
          <y>171</y>
          <width>321</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Тип продукта</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="foodBox"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QDateEdit" name="dateManufEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="addFoodButton">
             <property name="text">
              <string>Подтвердить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="quitAddFoodButton">
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
